src/darwin.rs

```diff
@@ -157,7 +130,7 @@ pub fn get_stats(kind: &StatType) -> Result<rusage, SporkError> {
     };
 
 
-    let mut usage = empty_rusage();
+    let mut usage = unsafe { std::mem::MaybeUninit::zeroed().assume_init() };
```

Comment:
We should not init `rusage` using `empty_rusage()`, since `rusage` contains extra private field while `target_env=musl` https://github.com/rust-lang/libc/blob/b76911ecbd94f02f750c425fd0e61ea89b4de012/src/unix/mod.rs#L122-L123